pkgpanda/util.py

```diff
@@ -26,12 +27,6 @@
 
 is_windows = platform.system() == "Windows"
 
-json_prettyprint_args = {
```

Comment:
Using this just once and putting the arguments into the `json_prettyprint` function is a purely stylistic change I decided to make.

Reply:
Well - with the new use of `write_string` within `write_json`, it became possible to use `json_prettyprint` and that meant that  `json_prettyprint_args` was used only in one place and could therefore be moved into the `json_prettyprint` function.